### PR TITLE
Add sample playbooks for vFTD deployment at KVM host.

### PR DESCRIPTION
**Pull request by 119Vik**
Add playbooks to explain the process of deployment vFTD to KVM host with Ansible.

### CODE REVIEW

File: samples/deployment/kvm/inventory.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+all:
+  vars:
+    ansible_user: "...username..."
+    ansible_ssh_private_key_file: "...ssh key path..."
+    ansible_become_pass: "...root passwd..."
+    ansible_connection: "ssh"
+    ftd_build_id: "6.4.0-1304"
+    ftd_vm_name: "vFTD-6.4"
+    main_folder: "/var/vm"
+    images_folder: "{{ main_folder }}/images"
```
[the remark this answers is not in the document]
Comment (119Vik) [reply samples/deployment/kvm/inventory.yaml]: fixed

File: samples/deployment/kvm/inventory.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    vms_folder: "{{ main_folder }}/kvm_hdds"
```
[the remark this answers is not in the document]
Comment (119Vik) [reply samples/deployment/kvm/inventory.yaml]: fixed

File: samples/deployment/kvm/inventory.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      - name: net1
+        subnet: 11
+      - name: net2
+        subnet: 12
+      - name: net3
+        subnet: 13
+  children:
+    kvm_hosts:
+      hosts:
+        ...kvm host name...:
```
[the remark this answers is not in the document]
Comment (119Vik) [reply samples/deployment/kvm/inventory.yaml]: no reason. removed it.

File: samples/deployment/kvm/setup_host.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          - libvirt-clients
+          - libvirt-daemon-system
+          - bridge-utils
+          - virt-manager
+      become: yes
+    - file:
+        path: "{{ images_folder }}"
+        state: directory
+        owner: "{{ ansible_user }}"
+      become: true
```
[the remark this answers is not in the document]
Comment (119Vik) [reply samples/deployment/kvm/setup_host.yaml]: default images folder path defined with this example will be  `/var/vm/images` which can't be created without root privileges.

File: samples/deployment/kvm/deploy_vm.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+- name: "Deploy vFTD at the KVM host"
```
Comment (annikulin) [inline samples/deployment/kvm/deploy_vm.yaml]: I guess, it should be `Deploy on` or `Deploy to`.
https://english.stackexchange.com/questions/5756/deployed-to-vs-deployed-on/5761

Comment (119Vik) [reply samples/deployment/kvm/deploy_vm.yaml]: fixed

**Pull request merged**